test/e2e/storage/persistent_volumes-local.go

```diff
@@ -400,6 +400,25 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 			err = framework.IssueSSHCommand(fileDoesntExistCmd, framework.TestContext.Provider, config.node0)
 			Expect(err).NotTo(HaveOccurred())
 
+			By("Deleting provisioner daemonset")
+			deleteProvisionerDaemonset(config)
+		})
+		It("should not create local persistent volume for filesystem volume that was not bind mounted", func() {
+
+			directoryPath := filepath.Join(config.discoveryDir, "notbindmount")
+			By("Creating a directory, not bind mounted, in discovery directory")
+			mkdirCmd := fmt.Sprintf("mkdir -p %v -m 777", directoryPath)
+			err := framework.IssueSSHCommand(mkdirCmd, framework.TestContext.Provider, config.node0)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Starting a provisioner daemonset")
+			createProvisionerDaemonset(config)
+
+			By("Waiting for a PersistentVolume to NOT be created for non bind mounted filesystem volume")
```

Comment:
Is it possible to check the logs of the provisioner container for the message?  Then the wait shouldn't need to be as long.  Try searching through TestContainerOutput for the API calls on how to get logs.